### ISSUE TITLE
typo

### ISSUE THREAD

**Issue by 0xsamgreen** [jcjohnson/pytorch-examples#16]
In the first example:

> Backprop to compute gradients of w1 and w2 with respect to loss

should be

> Backprop to compute gradient of loss with respect to w1 and w2

Thank you for the nice write-up!
